File: CryptChat/Base/Logic.cs
using CryptChat.Base.Commands;
using Telegram.Bot.Types;
using Telegram.Bot;

namespace CryptChat.Base
{
    public class Logic
    {
        public async Task StartAsync(Message message, TelegramBotClient client)
        {
            StartCommand start = new StartCommand();
            await start.Execute(message, client);
        }
    }
}

File: CryptChat.F#/Base/Logic.fs
namespace CryptChat.Base

open System.Threading.Tasks
open Telegram.Bot
open Telegram.Bot.Types

    type Logic() =
        member _.StartAsync(message: Message, client: TelegramBotClient) : Task =
            task {
                let str = "C R Y P T C H A T v. 1.0\n--------------------------------------\nКоманды для работы:\n/lastBlock - начать поиск по валюте"
                do! client.SendMessage(ChatId(message.Chat.Id), str) :> Task
            }

